_IBC/_middleware_ICS20_memo_keys.json

```diff
@@ -0,0 +1,77 @@
+{
+  "memo_keys": [
+    {
+      "key": "wasm",
+      "description": "To call a cosmwasm contract",
+      "git_repo": "https://github.com/osmosis-labs/osmosis/tree/main/x/ibc-hooks#ibc-hooks",
+      "memo": {
+        "wasm": {
+          "contract": "osmo1contractAddr",
+          "msg": {
+            "raw_message_fields": "raw_message_data"
+          }
+        }
+      }
+    },
+    {
+      "key": "ibc_callback",
+      "description": "Passing the ibc acknowledgement back to a cosmwasm smart contract",
+      "git_repo": "https://github.com/osmosis-labs/osmosis/tree/main/x/ibc-hooks#ibc-hooks",
+      "memo": {
+        "ibc_callback": "osmo1contractAddr"
+      }
+    },
+    {
+      "key": "autopilot",
+      "description": "One click liquid staking",
+      "git_repo": "https://github.com/Stride-Labs/stride/tree/main/x/autopilot",
+      "memo": {
+        "autopilot": {
+          "receiver": "strideXXX",
+          "claim": {
+            "stride_address": "strideXXX",
+            "airdrop_id": "evmos"
+          }
+        }
+      }
+    },
+    {
+      "key": "forward",
+      "description": "To send a packet from a source to destination chain when there is at least one chain in between, the intermediate chain uses the middleware, and next specifies the subsequent chain to route the packet to, after the chain specified in forward",
+      "git_repo": "https: //github.com/strangelove-ventures/packet-forward-middleware",
+      "memo": {
+        "forward": {
+          "receiver": "chain-c-bech32-address",
+          "port": "transfer",
+          "channel": "channel-123",
+          "timeout": "10m",
+          "retries": 2,
+          "next": {
+            "forward": {
+              "receiver": "chain-d-bech32-address",
+              "port": "transfer",
+              "channel": "channel-234",
+              "timeout": "10m",
+              "retries": 2
+            }
+          }
+        }
+      }
+    },
+    {
+      "key": "",
```

Comment:
Is having no key value going to be OK here? I assume we want keys to be unique, so shouldn't no value be off limits?

Reply:
I removed this entry for now, agree this is not strictly the format that should be used